backend/djangoindia/db/migrations/0008_remove_eventregistration_occupation_and_more.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.2.5 on 2024-08-28 17:35
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0007_sponsor_sponsorship'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='eventregistration',
```

Comment:
can you put underscore to distinguish, or alter the name and then use

Reply:
these migrations are auto generated .